### PR TITLE
[Feature] Support Asynchronous Inference/Eval of InternVLA-N1

### PR DESCRIPTION
**Pull request by steinate**
- Support the latest InternVLA-N1 model
- Support asynchronous evaluation & inference of dual-system models

### CODE REVIEW

File: internnav/model/utils/vln_utils.py [hunk 1]
```diff
@@ -121,6 +108,53 @@ def normalize_angle(angle):
 
         return actions
     
+    def reconstruct_trajectory(actions):
```
Comment (ZCMax) [inline internnav/model/utils/vln_utils.py:111]: Add the docstrings to describe the function

Comment (ZCMax) [reply internnav/model/utils/vln_utils.py:111]: Here is an example: 

        """Extract 3d features from the backbone -> fpn -> 3d projection.

        -> 3d neck -> bbox_head.

        Args:
            batch_inputs_dict (dict): The model input dict which include
                the 'imgs' key.

                    - imgs (torch.Tensor, optional): Image of each sample.
            batch_data_samples (list[:obj:`DetDataSample`]): The batch
                data samples. It usually includes information such
                as `gt_instance` or `gt_panoptic_seg` or `gt_sem_seg`.

        Returns:
            Tuple:
            - torch.Tensor: Features of shape (N, C_out, N_x, N_y, N_z).
            - torch.Tensor: Valid mask of shape (N, 1, N_x, N_y, N_z).
        """

File: scripts/eval/eval_habitat.py [hunk 2]
```diff
@@ -58,7 +58,9 @@ def main():
     np.random.seed(local_rank)
     
     #* 1. Load model and tokenizer. Currently, we support two modes: dual_system and system2 in Habitat.
-    processor = AutoProcessor.from_pretrained(args.model_path)
+    processor = AutoProcessor.from_pretrained("Qwen/Qwen2.5-VL-7B-Instruct")
```
Comment (ZCMax) [inline scripts/eval/eval_habitat.py:61]: Why do we need to use the processor of Qwen2.5-VL?

File: internnav/habitat_extensions/evaluator.py [hunk 3]
```diff
@@ -623,10 +624,39 @@ def eval_action(self, idx) -> None:
                             action = action.detach().cpu().numpy()[0] if isinstance(action, torch.Tensor) else action
                             action = action[0] if hasattr(action, "__len__") else action
                         else: # dual-system logic
-                            if len(local_actions) != 0:
+                            if len(local_actions)==0:
+                                ############nav_dp###########
+                                local_actions = []
+                                image_dp = torch.tensor(np.array(look_down_image.resize((224, 224)))).to(torch.bfloat16) / 255
```
Comment (ZCMax) [inline internnav/habitat_extensions/evaluator.py:630]: Generally, we do not use hard code. It's better to change it into a parameter, or leave the comments to explain why to resize it into 224.
